docs/io/unified.rst

```diff
@@ -318,6 +327,78 @@ the keywords ``DATE``, ``DATE-*`` (ISO-8601 datetime strings) and the ``MJD-*``
 For more details regarding the FITS time paper and the implementation,
 refer to :ref:`fits_time_column`.
 
+Reading time columns in non-Astropy written FITS tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Reading FITS files which are compliant with the current FITS time standard,
+that is, the files written after the standard was set in stone, is supported
+by Astropy by following the rules and conventions set by the standard. Thus,
+time coordinate columns in arbitrary FITS files (compliant with the standard)
+will be read as native `~astropy.time.Time` objects.
+
+However, a major chunk of the astronomical datasets, most of which are dated
```

Comment:
"However, many astronomical data files, including official data products from major observatories, follow different conventions that pre-date the FITS standard and do not strictly adhere to it."